webapp/src/route.tsx

```diff
@@ -30,7 +30,8 @@ function FBRoute(props: RouteProps) {
     const clientConfig = useAppSelector<ClientConfig>(getClientConfig)
 
     let redirect: React.ReactNode = null
-    const disableTour = clientConfig?.featureFlags?.disableTour || false
+    // No FTUE for guests
+    const disableTour = me?.is_guest || clientConfig?.featureFlags?.disableTour || false
```

Comment:
@harshilsharma63 do you have any concerns skipping the FTUE for guests this way?